lib/commands/project/create.ts

```diff
@@ -37,9 +37,9 @@ export class CreateCommand extends ProjectCommandBaseLib.ProjectCommandBase {
 	}
 
 	public canExecute(args: string[]): IFuture<boolean> {
-		return this.canExecuteCore();
+		return this.canExecuteCore(args[0]);
```

Comment:
what will happen in case you write "$ appbuilder create app1 app2 app3"? This should not be allowed, but based on this canExecute method we'll use only the first parameter, instead of saying that parameters app2 and app3 are not valid for this command.